h2o-hadoop-2/h2o-hdp2.6-assembly/build.gradle

```diff
@@ -1,6 +1,7 @@
 ext {
   hadoopVersion = 'hdp2.6'
-  hadoopMavenArtifactVersion = '2.7.3.2.6.1.0-129'
+  hadoopMavenArtifactVersion = '2.7.3.2.6.5.196-1'
+  hdfsDependency = "org.apache.hadoop:hadoop-hdfs"
```

Comment:
this should be just `hadoop-hdfs` no?
also maybe it deservers a comment?

Reply:
thanks for catching that, I will fix